Formula/easy-rsa.rb

```diff
@@ -37,6 +42,6 @@ def install
     doc.install "README.quickstart.md"
 
     bin.install_symlink share/"easyrsa" => "easyrsa"
-    share.install share/"vars.example" => "vars"
+    # share.install share/"vars.example" => "vars.example"
```

Comment:
Please install `vars.example` to `vars`

Reply:
Does it really needed? I'm patching easy-rsa script to have correct values for `EASYRSA_OPENSSL` and `EASYRSA`. So there is no need to actually override them with `vars` (I'm patching vars.example with the same values for consistency).

Also lots of guides on how to use easy-rsa refer to vars.example. I think it will be better to have that file.